src/components/include/jupyterlite/BaseJupyterLiteComponent.tsx

```diff
@@ -86,6 +78,18 @@ class BaseJupyterLiteSessionComponent<P = never, S = never> extends React.Compon
         }
     };
 
+    // eslint-disable-next-line react/sort-comp
+    messageHandlerConfigs: IMessageHandlerConfigItem[] = [
+        {
+            action: "set-data",
```

Comment:
Let's add an enum in https://github.com/Exabyte-io/cove.js/blob/37c8233d576b0260ae07618aa2ce879eee6a8d3b/src/other/iframe-messaging/IframeToFromHostMessageHandler.ts#L10C39-L10C81 and reuse here

```
import actionsEnums
actionsEnums.setData
```